mempool/src/pool/entry.rs

```diff
@@ -53,6 +53,10 @@ impl TxEntry {
     pub fn size(&self) -> usize {
         serialization::Encode::encoded_size(self.transaction())
     }
+
+    pub fn outpoints(&self) -> impl '_ + Iterator<Item = &OutPoint> + ExactSizeIterator {
+        self.transaction().inputs().iter().map(|i| i.outpoint())
```

Comment:
In Gosha's incoming PR, this will not be possible. We'll have accounts too, which won't have outpoints, but account + nonce.